main.go

```diff
@@ -14,6 +15,82 @@ import (
 
 const Version = "v0.4"
 
+type EC2InstanceType struct {
+	Name                 string
```

Comment:
We may as well omit the Name property, as it isn't used, and is just duplicated data.